mongofiles/options.go

```diff
@@ -102,6 +103,10 @@ type StorageOptions struct {
 	// By default, mongofiles waits for a majority of members from the replica set to respond before returning.
 	// Cannot be used simultaneously with write concern options in a URI.
 	WriteConcern string `long:"writeConcern" value-name:"<write-concern>" default-mask:"-" description:"write concern options e.g. --writeConcern majority, --writeConcern '{w: 3, wtimeout: 500, fsync: true, j: true}'"`
+
+	// RegexOptions specifies the options passed to "$regex" queries that are used for get_regex
+	// The default is to use no options, i.e. standard PCRE syntax
+	RegexOptions string `long:"options" value-name:"<regex-options>" short:"o" description:"regex options used for get_regex"`
```

Comment:
if this is a common option, which means it can be put anywhere in the command, such as `mongofiles --options i get_regex. ` Maybe we should rename it to some more unambiguous name such as `get-regex-options`?
Or we should check this options to be positioned after get_regex subcommand.

Reply:
I don't think either of those are necessary. The comments indicate that `StorageOptions` contains all the mongofiles-specific command-line options, being that that's also where `--local` is specified.

Like I mentioned, I followed the precedent set by the implementation of `get ... --local` for optional command-line flags specific to mongofiles. Unless I'm mistaken, that precedent indicates that no additional checks on our side are necessary.

Reply:
On second thought, I'd be fine with renaming it to `--regex-options` (even though I'm not sure if it's necessary), as long as it remains consistent with the implementation of `--local`.

Reply:
local and options have a pretty big difference to the users. I am for renaming this field. But I'd like to hear others' thoughts.

Reply:
The commands use snake case but the options use camel case (see docs). So it should be something like: 

```
mongofiles get_regex <pattern> --regexOptions <options>
```

I don't know why it's different in commands vs options, but we should maintain consistency with what's already there.

Reply:
This sounds good to me!